tests/integration/modules/mac_service.py

```diff
@@ -70,17 +63,18 @@ def test_launchctl(self):
         '''
         # Expected Functionality
         self.assertTrue(
-            self.run_function('service.launchctl', ['error', 'bootstrap', 64]))
+            self.run_function('service.launchctl',
+                              ['error', 'bootstrap', '64']))
         self.assertEqual(
             self.run_function('service.launchctl',
-                              ['error', 'bootstrap', 64],
+                              ['error', 'bootstrap', '64'],
```

Comment:
@cachedout Poss because `args` are expected to be strings? The `service.launchctl` function accepts `sub_cmd`, `*args` and `**kwargs` as parameters. I'm assuming `*args` must be strings?

If I pass an integer on the command line with the following command:
```
salt-call --local service.launchctl error bootstrap 64
```
I get the following error:
```
Passed invalid arguments: sequence item 3: expected string, int found.
```
If I pass it as a string (`'"64"') the command functions as expected.

In the test suite, you apparently pass a string by enclosing in single quotes (`'64'`)